sktime/performance_metrics/tests/test_performance_metrics_forecasting.py

```diff
@@ -368,6 +367,25 @@
 }
 
 
+def _call_metrics(loss_func, loss_class, y_true, y_pred, y_train, y_pred_benchmark):
```

Comment:
```suggestion
def _call_metric(metric_func, metric_class, y_true, y_pred, y_train, y_pred_benchmark):
```

Reply:
Updated in the function and also changed "loss" to "metric" throughout `test_performance_metrics_forecasting.py`